agent/lib/ipaddr_helpers.rb

```diff
@@ -0,0 +1,48 @@
+class IPAddr
```

Comment:
We're doing IPAddr patching in many places now, maybe it's time to roll these as a separate gem? (not maybe part of this PR, but overall)